poi.go

```diff
@@ -467,13 +467,14 @@ func toBranch(branchNames []string) []Branch {
 
 	for _, branchName := range branchNames {
 		splitedNames := strings.Split(branchName, ",")
-		head := false
-		if splitedNames[0] == "*" {
-			head = true
-		}
+
+		headValue := splitedNames[0]
+		refNameValue := strings.Join(splitedNames[1:len(splitedNames)-1], ",")
```

Comment:
This approach will not work when the number of information to be retrieved with `--format` is increased to four (e.g. `--format=%(HEAD),%(refname:lstrip=2),%(objectname),%(whatever)`).
Another idea is to use `--format` to change to a delimiter such as a whitespace that cannot be used in branch names

Reply:
I was a little nervous about including spaces in the command line argument, so I changed it to use a colon, which is also not included in the branch name.

(You can check that you can't use colons with `man git-check-ref-format`, but also when I actually try to created it, it created a branch name with the colon part missing.)

ea56c1727bcde8cf48d3177ce817690174c6ba21

Reply:
>I was a little nervous about including spaces in the command line argument

I think I can enclose `--formt` in quotation marks. But this alone doesn't seem to work. It seems like it would be good to make it a `:`.
`"--format='%(HEAD) %(refname:lstrip=2) %(objectname)'"`